docs/simulation.md

```diff
@@ -1,15 +1,29 @@
 ## Simulation
 
 Cirq comes with built in Python simulators for testing out
-small circuits.  One of these simulators works for generic
-gates that implement their unitary matrix, ``cirq.Simulator``.
+small circuits.  The two main types of simulations that Cirq
+supports are pure state simulators and mixed state simulators.
+For pure state simulations there are two variations of simulators.
+One of these pure state simulators works for generic
+gates that implement their unitary matrix: ``cirq.Simulator``.
 The other simulator is customized for the native gate set
 of Google's Xmon hardware ``cirq.google.XmonSimulator``.
 This later simulator can shard its simulation  across
 different processes/threads and so take advantage of
 multiple cores/CPUs.  Depending on your local computer
 architecture one or the other of these may be faster, but
-we recommend starting with ``cirq.Simulator``.
+we recommend starting with ``cirq.Simulator``.  Mixed state
+simulators are supported by ``cirq.DensityMatrixSimulator``.
+
+In general the name *pure state simulator* and *mixed state
+simulators* refers to the fact that these simulations are
+for quantum circuits, including unitary, measurements, and noise
```

Comment:
Something about "including unitary" is a bit off to me. Is this saying that these simulators simulate unitary gates?